index.js

```diff
@@ -26,13 +26,16 @@ Dcdr.prototype.init = function(config) {
 Dcdr.prototype.watchConfig = function() {
   fs.watch(this.config.dcdr.path, { persistent: false }, function() {
     this.config.logger.info('Reloading features from ' + this.config.dcdr.path);
-    this.loadFeatures(this.config.dcdr.path);
+    this.loadFeatures(false, this.config.dcdr.path);
   }.bind(this));
 };
 
-Dcdr.prototype.loadFeatures = function(path) {
-  var features = JSON.parse(fs.readFileSync(path, 'utf8'));
-  this.setFeatures(features);
+Dcdr.prototype.loadFeatures = function(isInitialLoad, path) {
```

Comment:
Would it be better to do the initial load as a new optional second argument in order to allow us to cut a minor release as opposed to a breaking change to the function signature?

Reply:
Oh, I got confused about what constitutes the declared API. Fixed the parameter order.

Reply:
Ah it might not be declared officially, but since it could be accessed even unintentionally I thought it might be worth a change.